src/index.html

```diff
@@ -11,7 +11,198 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset class="fieldsets">
+        <legend>Personal information</legend>
+
+        <label class="form-field">
+        Surname:
+          <input
+              type="text"
+              name="surname"
+              required
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input
+              type="text"
+              name="name"
+              required
+          >
+        </label>
+
+        <div class="form-field">
+          <label>
+           How old are You?
+            <input
+              type="number"
+              name="age"
+              value="12"
+              min="1"
+              max="100"
+              required
+            >
+          </label>
+        </div>
+
+        <label class="form-field">
+              Full date of birth:
+          <input
```

Comment:
The same, fix in all cases

```suggestion
        <label class="form-field">
          Full date of birth:
          <input
```